### DISCUSSION TITLE
Can't save the matrices.

### DISCUSSION THREAD

**Discussion by Shubham-Kr-Shaw** [pycaret/pycaret#1377] [Q&A] [answered]
How to save or store the matrices that we get in form of the table after running " compare_models() ".

**Comment by pycaret** [comment, marked as answer]
After compare_models just use the function pull to store that grid into pandas DataFrame.

best = compare_models()
best_results = pull()

Hope this helps.

**Reply by Shubham-Kr-Shaw** [reply]
Thanks a lot.
I am in love with Pycaret.

Regards 
Shubham Kr Shaw
